domains/mrvasil.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+      "username": "mrvasil",
+      "email": "lol@urrr.ru"
+    },
+    "record": {
+        "URL": "http://server.silaeder.ru:12360"
```

Comment:
I cannot reach you website, keeps on loading

Reply:
Sorry, its not available yet, but it'll be up soon :(
Please connect the domain

Reply:
hohoho xd slitik zabombilo